### PR TITLE
issue-1

### PR DESCRIPTION
**Pull request by Anush17**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 #editors
 .idea
+bower_components
```
Comment (Mariam13) [inline .gitignore]: @aaa17 please leave the empty line

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "jQuery-Mention",
+  "homepage": "https://github.com/Mariam13/jQuery-Mention",
+  "authors": [
```
Comment (Mariam13) [inline bower.json]: @aaa17 please add author with name and email

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "Anush  <achakhoyan99@gmail.com>"
+  ],
```
Comment (Mariam13) [inline bower.json]: @aaa17 please add contributors

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  "description": "",
+  "main": "",
+  "license": "MIT",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
```
Comment (Mariam13) [inline bower.json]: @aaa17 please add bugs url

File: index.html [hunk 5]
```diff
@@ -1,16 +1,18 @@
...
-    <div>
-        <textarea class="mention-box" rows="" cols=""></textarea>
-    </div>
+<div>
+    <textarea class="mention-box" rows="" cols=""></textarea>
+</div>
 
+
+<script type="text/javascript" src="js/mention-script.js"></script>
+<script src="bower_components/jquery/dist/jquery.min.js"></script>
```
Comment (Mariam13) [inline index.html]: @aaa17 please add this two scripts in the header, and first of then must be jquery

Comment (Anush17) [reply index.html]: but I know that script fils always will be under the body ?

File: .gitignore (hunk 1 above)
Comment (Anush17) [reply .gitignore]: before #editors ?

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,4 @@
 #editors
+
 .idea
+bower_components
```
Comment (Mariam13) [inline .gitignore]: @aaa17 please add empty line

File: bower.json [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  "bugs": {
+    "issue-1": "https://github.com/Mariam13/jQuery-Mention/issues/1",
+    "issue-2": "https://github.com/Mariam13/jQuery-Mention/issues/1",
+    "issue-3": "https://github.com/Mariam13/jQuery-Mention/issues/1"
```
Comment (Mariam13) [inline bower.json]: @aaa17 not all busg, just link for issue's

File: index.html [hunk 8]
```diff
@@ -1,16 +1,17 @@
 <!DOCTYPE>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
     <title>Jquery Mention | Plugin</title>
-    <link rel="stylesheet" type="text/css" href="css/mention-styles.css" />
+    <link rel="stylesheet" type="text/css" href="css/mention-styles.css"/>
     <script type="text/javascript" src="js/mention-script.js"></script>
+    <script src="bower_components/jquery/dist/jquery.min.js"></script>
```
Comment (Mariam13) [inline index.html]: @aaa17 please move the jQuery script top of our mention.ts

**Pull request merged**